spec/fixtures/diversion_policy.yml

```diff
@@ -13,3 +13,6 @@
 dp_1:
   id: 1
   name: Clear header
+dp_2:
+  id: 2
+  name: Clear
```

Comment:
this is constant value

do not add new value to fixture if it was not explicitly said in an issue